packages/react/src/Spotlight/Spotlight.test.tsx

```diff
@@ -48,4 +48,10 @@ describe('Spotlight', () => {
 
     expect(component).toHaveClass('amsterdam-spotlight--green')
   })
+
+  it('renders a custom tag', () => {
+    render(<Spotlight as="article" />)
```

Comment:
Spotlight should be named in the PR title for future reference.

Reply:
Spotlight already supported this and its API hasn’t changed. What would you like to mention in the PR title?

Reply:
Allow ‘Grid Cell’ and 'Spotlight' to render a custom HTML element

Reply:
No, Spotlight didn’t change here. It had an `as` prop from the start in #636, you created it yourself.